docs/app/css/style.css

```diff
@@ -51,6 +51,31 @@ ul li:first-child {
   margin-top: 0;
 }
 
+ul.skip-links li {
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+ul.skip-links li a {
+  background-color: #fff;
+  display: block;
+  font-size: 1.25em;
+  margin: 0.5em;
+  opacity: 0;
```

Comment:
The pattern for skip links is to have them always render but show visibly on focus. That way they are in the page for screen reader users at all times.